docs/integration/authorization.adoc

```diff
@@ -224,7 +224,7 @@ The result token is a base64 encoded JSON object including the following paramet
 |=======================================================================================================================================================
 |Name |Type |Description
 |account |String |The unique id of the user account on agrirouter , that will be provided to you in the metrics exports for billing
-|regcode |String |If response-type=onboard, this regcode will deliver a Registration Code equal to clicking the generate TAN-Button in the agrirouter ui
+|regcode |String |If response-type=onboard, this regcode will deliver a Registration Code equal to clicking the generate RegistrationCode-Button in the agrirouter ui
```

Comment:
From my point of view - there"s only a "Telemetry-Connection" button, which adds a telemetry connection and then generates a registration code. Could be more precise.

Reply:
Wording adjusted